persistence.js

```diff
@@ -29,6 +29,7 @@ function matchTopic (p) {
 }
 
 MemoryPersistence.prototype.storeRetained = function (packet, cb) {
+  packet = Object.assign({}, packet)
```

Comment:
Using aedes-packet is breaking tests, as brokerId is  used instead of just id and it also enforces messageId